DungeonAndDragonsRollStats/src/main/enums/CharacterRace.java

```diff
@@ -0,0 +1,91 @@
+package main.enums;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum CharacterRace {
+	Dragonborn(Arrays.asList()),
+	Dwarf(Arrays.asList(

```

Comment:
Let's talk about this sometime in the future, I am still new to the character/subcharacter stuff and not sure if you can combine two characters (although this could be in the same thought as your earlier concern regarding custom abilities).